server/src/controllers/api/users.js

```diff
@@ -176,21 +169,47 @@ class UsersApiController {
   static addDonation(req, res, next) {
     const { email } = req.decoded;
 
-    req.body.amount = 5;
-
     User.findOneAndUpdate(
       { email: email.trim().toLowerCase() },
       { $push: { donations: req.body } },
       (err) => {
         if (err) return next(err);
 
-        res.json({
+        res.status(201).json({
```

Comment:
I noticed you're using the "findOneAndUpdate" method which is used to update an entry in the database but your response status code is 201 which is the response code to use when a new entry is created. Is there a reason why you are returning a 201 status code instead of a 200 status code ?.

Reply:
Yes. I did that because it updates the User collection but actually creates a new Donation. The Donations array is part of the User Collection. So 200 didn't seem right since it's a creation. Tomorrow, during our TTL interaction, the team would make a decision on which status code is the better for this case.

Reply:
Alright. Brilliant decision.

Reply:
We decided to go with 200 and I have updated it. Thank you